### PR TITLE
feat(feedback): add config for feedback DLQ

### PR DESCRIPTION
**Pull request by aliu39**
Need to verify the values of `dlq_` fields in consumers/__init__. These are used to create a DlqPolicy (line 509, same file)

requires https://github.com/getsentry/sentry-kafka-schemas/pull/238

### CODE REVIEW

File: src/sentry/consumers/__init__.py [hunk 1]
```diff
@@ -267,6 +267,9 @@ def ingest_events_options() -> list[click.Option]:
         "static_args": {
             "consumer_type": "feedback-events",
         },
+        "dlq_topic": Topic.INGEST_FEEDBACK_EVENTS_DLQ,
+        "dlq_max_invalid_ratio": 0.01,
+        "dlq_max_consecutive_count": 1000,
```
Comment (lynnagara) [inline src/sentry/consumers/__init__.py:272]: Mind removing these 2 lines? We experimented with various values but now I think we should just skip these (across the board not specific to your use case)

Comment (aliu3ntry) [reply src/sentry/consumers/__init__.py:272]: Sure, should we remove from the other ones too?

Comment (lynnagara) [reply src/sentry/consumers/__init__.py:272]: Yes we probably should. But I'll double check and do it as a follow up -- let's just focus on the ingest-feedback-events-dlq here.

Comment (aliu3ntry) [reply src/sentry/consumers/__init__.py:272]: yep, sounds good

Comment (aliu3ntry) [reply src/sentry/consumers/__init__.py]: similarly, I noticed ingest-events consumer doesn't have a dlq_topic. Not something to worry about for this PR though

Comment (lynnagara) [reply src/sentry/consumers/__init__.py]: I'm pretty sure it does have now. I just added it. You might need to rebase/merge master to see it.

File: tests/sentry/conf/test_kafka_definition.py [hunk 2]
```diff
@@ -16,8 +16,6 @@ def test_topic_definition() -> None:
     # TODO: Remove this once these topics are actually registered in sentry-kafka-schemas
     currently_unregistered_topics = [
         "outcomes-billing",
-        "ingest-events",
-        "ingest-events-dlq",
```
Comment (lynnagara) [inline tests/sentry/conf/test_kafka_definition.py:20]: thanks!

**Review by lynnagara — APPROVED** [review]

**Pull request merged**